runtime/tests/third_party/dummy_device.toml

```
@@ -91,6 +58,38 @@ matrix = [
         "MultiControlledX",
 ]
 
+[operators.gates.named]
```

Let's stick with the name
```suggestion
[operators.gates.native]
```

---

Done